src/workerd/jsg/jsg.h

```diff
@@ -15,6 +15,7 @@
 #include <workerd/jsg/memory.h>
 #include <workerd/util/strong-bool.h>
 
+#include <kj-rs/convert.h>
```

Comment:
do you need convert or rust/cxx.h will be enough?

Reply:
Yeah rust/cxx.h is enough. Nice catch.